docs/roman/references_general/flat_reffile.inc

```diff
@@ -20,36 +20,26 @@ the following keywords are *required* in FLAT reference files,
 because they are used as CRDS selectors
 (see :ref:`flat_selectors`):
 
-=========  ==============================  =============================
-Keyword    Data Model Name                 Instruments
-=========  ==============================  =============================
-DETECTOR   model.meta.instrument.detector  All
-EXP_TYPE   model.meta.exposure.type        
-FILTER     model.meta.instrument.filter
-BAND       model.meta.instrument.band      
-READPATT   model.meta.exposure.readpatt    
-GRATING    model.meta.instrument.grating   
-=========  ==============================  =============================
+===============   ===============================  ==============
+Attribute           Fully qualified path             Instruments
+===============   ===============================  ==============
+detector           model.meta.instrument.detector   WFI
+exptype            model.meta.exposure.type
+optical_element    model.meta.instrument.filter
+===============   ===============================  ==============
 
 Reference File Format
 +++++++++++++++++++++
-FLAT reference files are ASDF format, with 3 IMAGE arrays
-and 1 table. The format and content of the file is as follows:
-
-=======  ========  =====  ==============  ==============================
-EXTNAME  XTENSION  NAXIS  Dimensions      Data type
-=======  ========  =====  ==============  ==============================
-SCI      IMAGE       2    ncols x nrows   float
-ERR      IMAGE       2    ncols x nrows   float
-DQ       IMAGE       2    ncols x nrows   integer
-DQ_DEF   TABLE       NA   4 columns       integer, integer, string, string
-=======  ========  =====  ==============  ==============================
-
-The ``DQ_DEF`` table extension lists the bit assignments for the flag conditions
-used in the DQ array.
-
-.. include:: ../includes/dq_def.inc
-
-The ASDF file contains a single set of SCI, ERR, and DQ arrays, and a DQ_DEF
-table.  Image dimensions should be 2048x2048.
-
+FLAT reference files are ASDF format, with 3 data arrays.
+The format and content of the file is as follows:
+
+=======  ============ =====  ==============  =============
+Data      Array Type  NAXIS    Dimensions      Data type
+=======  ============ =====  ==============  =============
+data      NDArray         2    ncols x nrows    float
+err       NDArray         2    ncols x nrows    float
+dq	      NDArray         2    ncols x nrows    integer
+=======  ============  =====  ==============  ============
+
+The ASDF file contains a single set of data, err, and dq arrays.
+The array dimensions should be 4096x4096.
```

Comment:
If the array dimensions are known, we may put them in the table instead of `ncols x nrows`.
I'm not sure we need the `NAXIS` column. It's a FITS keyword.
And should we be more specific about the data type, as in `float32` and `uint16`?